src/js/disability-benefits/components/ClaimEstimationPage.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Link } from 'react-router';
```

Comment:
Maybe? For now, let's consider that a separate issue, since this change can go out before the switch to all claims.